fingerprint.c

```diff
@@ -66,6 +66,17 @@ void *enroll_thread_loop(void *arg)
     while((status = fpc_capture_image(sdev->fpc)) >= 0) {
         ALOGD("%s : Got Input status=%d", __func__, status);
 
+        pthread_mutex_lock(&sdev->lock);
+        if (!sdev->worker.thread_running ) {
+            pthread_mutex_unlock(&sdev->lock);
+            ALOGI("%s : finishing",__func__);
+            return NULL;
+        }
+        pthread_mutex_unlock(&sdev->lock);
+
+        if(status >= 1000)
```

Comment:
Magic constant.

Reply:
I'm sorry, but I don't get this. :(

Reply:
@rudcode He means there's no explanation what `1000` represents. The original problem stems from https://github.com/sonyxperiadev/vendor-sony-oss-fingerprint/blob/837e772e17a07dfd51a92f1b321ad3ea09076c57/fpc_imp_loire_tone.c#L440

which doesn't have an explanation either...

Reply:
1000 returned when finger is still not lost, so when finger is still not lost just continue, I still don't see what's wrong

Reply:
The issue is when someone reads the code, 1000 means literally nothing. So I want you to use a #define for it
I guess as @clshortfuse says, the function returning this status should have used a #define in the first place.